src/main/java/cz/cvut/kbss/study/security/SecurityConstants.java

```diff
@@ -27,7 +27,32 @@ private SecurityConstants() {
      */
     public static final int SESSION_TIMEOUT = 12 * 60 * 60;
 
-    public static final String ROLE_USER = "ROLE_USER";
+    public static final String user = "user";
+
+    public static final String administrator = "administrator";
```

Comment:
```suggestion
    public static final String administrator = "ROLE_ADMIN";
```

Reply:
I thought that with this iteration we want to keep same string. So nothing would change when deploying it.

Reply:
I removed it due to refactoring out Security Roles.

Reply:
yes ... since ROLE_ADMIN is only temporary i do not see advantage of refactoring it.

Reply:
ok, I got you

Reply:
I would even put it back to:
`public static final String ROLE_USER = "ROLE_USER";`

I just do not want to make extra commit to rename and remove it anyway as it won't be that trackable.

Does it make sense?

Reply:
I should rename it back in every rest endpoint. 
(I mean this):
```java
@PreAuthorize("hasAuthority('" + SecurityConstants.administrator + "'))
```
Anyway, we plan to delete these in the next iteration, so it doesn’t make much sense to me...

Reply:
The plan to delete it in the next iteration is exactly the reason why I did not want to rename it.

If it is not easy to rename, it is not worth the effort, because we will have clean history.  (I would like to squash most of the commits at the end so we can see the actual changes more clearly.) It is already quite challenge-ing to do the review.

If it takes more than 5 mins, leave it.

Reply:
Done, I renamed it.